wasm-scripts/pre.js

```diff
@@ -13,8 +13,8 @@ Module.preRun = () => {
 
 Module.onRuntimeInitialized = () => {
   const emnapi = Module.emnapiInit({
-    context: require('@tybys/emnapi-runtime').createContext(),
-    // nodeBinding: require('@tybys/emnapi-node-binding')
+    context: require('@emnapi/runtime').getDefaultContext(),
+    // nodeBinding: require('@emnapi/node-binding')
```

Comment:
Btw, it's a bit unclear from the docs - how would I know if I need those Node bindings? It seems to work even w/o them, even though Sharp does use Node-specific async APIs...

Reply:
`@emnapi/node-binding` is needed only if user is using the following things:

- `napi_async_init` / `napi_async_destroy` / `napi_make_callback`, otherwise they always return `napi_generic_failure`.
- The `async_resource` parameter of `napi_create_async_work` and `napi_create_threadsafe_function`, otherwise they don't work with `async_hooks`